chia/protocols/shared_protocol.py

```diff
@@ -5,7 +5,7 @@
 from chia.util.ints import uint8, uint16
 from chia.util.streamable import Streamable, streamable
 
-protocol_version = "0.0.32"
+protocol_version = "0.0.33"
```

Comment:
do we record which messages are available in what version of the protocol anywhere?

Reply:
No yet